SUMMARY.md

```diff
@@ -0,0 +1,19 @@
+## Table of Contents
+
+* [Connecting to Database](content/ContectingToDatabase/ConnectingToDatabase.q.md) ** - minutes**
+    > In this lesson, you will learn how to use PHP to establish a connection with PostgreSQL.
+
+* [PHP Data Objects](content/PDOClasses/PDOClasses.q.md) ** - minutes**
+    > In this lesson, you will learn about PHP data object (`PDO`) and how to run queries using `PDO::query`.
+
+* [Prepared Statement with execute()](content/UsingExecute/UsingExecute.q.md) ** - minutes**
+    > Discover how you can execute prepared statements using `PDOStatement::execute` with binding methods such as `bindParam()` and `bindValue()`.
+
+* [Prepared Statement with fetch()](content/UsingFetch/UsingFetch.q.md) ** - minutes**
+    > Learn how you can use `fetch()` to get data out of a prepared statement. You will also learn how to use different kinds of fetch modes in PDO.
+
+* [Prepared Statement with fetchAll()](content/UsingFetchAll/UsingFetchAll.q.md) ** -minutes**
+    > Discover how you can get a plain array, column, key pair values, row indexed by unique field, and row grouped by some field using `PDOStatement:fetchAll()`.
+
```

Comment:
I think the following are needed. 
 - `fetchColumn()`
 - `columnCount()`
 - `rowCount()`
 - `nextRowSet()`


Please check if which part/lesson do we have to include them or is it advisable to add another lesson? Thanks